sscanss/app/dialogs/insert.py

```diff
@@ -1204,6 +1205,23 @@ def updateCrossSection(self):
         self.showBounds(self.bounds_button.isChecked())
         self.clearShape()
 
+    def highlightPoints(self, rows):
+        """Highlights the points corresponding to the rows currently selected in the point manager
+        :param rows: the currently highlighted rows
+        :type rows: List[bool]
+        """
+        items = self.scene.items()
+        fixed_points = [item for item in items if isinstance(item, GraphicsPointItem) and item.fixed]
+
+        for point, row in zip(fixed_points, reversed(rows)):
```

Comment:
This assumes the scene point order is always the same as the rows order which is not right